_posts/2017-11-13-tetris-ai-experiment-1-2-single-parent-evolutionary-algorithm.md

```diff
@@ -0,0 +1,258 @@
+---
+author: Dan Cohen
+date: 2017-11-13 08:00:00 +00:00
+layout: post
+asset-type: post
+slug: tetris-ai-single-parent-evolutionary-algorithm
+title: "Tetris AI, Experiments 1 & 2: Single Parent Evolutionary Algorithm"
+canonical:
+    name: hacking dan
+    href: http://blog.hackingdan.com/2017/11/tetris-ai-experiment-1-single-parent.html
+image:
+    src: /assets/img/custom/blog/2017-11-13-tetris-ai-experiment-1-2-single-parent-evolutionary-algorithm/neural-network.png
+tags:
+- evolutionary algorithm
+- machine learning
+- artificial intelligence
+- neural network
+---
+
+## Abstract (why not make it formal)
+
+In this experiment, I implement an evolutionary algorithm with no crossover to evolve a neural network with the intention of having it learn to play tetris. I assess the results of my evolutionary algorithm applied to learning tetris. The results show the impact of a high occurrence of mutations(χ) and a high standard deviation for mutations(σ)  
```

Comment:
Would be cool to have a screenshot of the visualization here, e.g. show the thing playing tetris